doc/whats_new.rst

```diff
@@ -17,6 +17,8 @@ Changelog
 
     - Added the ability to decimate :class:`mne.Evoked` objects with :func:`mne.Evoked.decimate` by `Eric Larson`_
 
+    - Add 'cmap=interactive' option to plotting functions by `Jaakko Leppakangas`_
```

Comment:
double-backticks on the quote will read nicer, and `interactive` should probably be a string in that code bit